plugin.json

```
@@ -54,11 +54,17 @@
                 "type": "longtext",
                 "help_text": "The contents of a E20 license."
             },
+            {
+                "key": "GroupID",
+                "display_name": "Group ID",
+                "type": "text",
+                "help_text": "(Optional) When set, all installations created by the plugin are created within this group."
```

```suggestion
                "help_text": "(Optional) When set, all new installations created by the plugin are created within this group."
```

Worth a slight change here to indicate that if you add a group ID after installations exist it will only be used for new installations?

---

Yep, good point.